src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24105.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue24105 : _IssuesUITest
+	{
+		public override string Issue => "CarouselView doesn't scroll to the right Position after changing the ItemSource with Loop enabled";
+
+		public Issue24105(TestDevice testDevice) : base(testDevice) { }
+		
+		[Test]
+		[Category(UITestCategories.CarouselView)]
+		public void CarouselViewShouldScrollToRightPosition()
```

Comment:
The test fails on Windows, at the end, maybe the apps crashed?
`The app was expected to be running still, investigate as possible crash`
While in Android, is also failing but because the reference snapshot changed:
![image](https://github.com/user-attachments/assets/15771d76-8fd3-47cd-94da-851569dd8646)
The item text alignment changed and is aligned at the end. Looks incorrect based on the HorizontalTextAlignment property.